scripts/run-sonar-scanner-cli.sh

```diff
@@ -21,15 +21,22 @@ if [[ -n "${INPUT_PROJECTBASEDIR}" ]]; then
   scanner_args+=("-Dsonar.projectBaseDir=${INPUT_PROJECTBASEDIR}")
 fi
 
+# The SSL folder may exist on an uncleaned self-hosted runner
+SONAR_SSL_FOLDER=~/.sonar/ssl
+if [ -d "$SONAR_SSL_FOLDER" ]; then
```

Comment:
We clean even when `SONAR_ROOT_CERT` is not defined, to avoid situations where a previous execution of the GitHub action done with `SONAR_ROOT_CERT` has left a certificate in `~/.sonar/ssl`, which would be picked up by the Sonar Scanner CLI at the next execution.

Reply:
What if users want to prepare their self-hosted agents by placing already a `truststore.p12` or `keystore.p12` at the right location?
Can't we instead remove the certificate with alias "sonar" first: https://superuser.com/questions/881665/keytool-commands-to-replace-existing-ssl-certificate

I don't think there is a problem if multiple runs reuse the same truststore.p12. If people really want to start with a clean truststore.p12, they could tune their agent I think.